src/Algebra/Consequences/Propositional.agda

```diff
@@ -26,17 +26,17 @@ import Algebra.Consequences.Setoid (setoid A) as Base
 
 open Base public
   hiding
-  ( comm+assoc⇒middleFour
-  ; identity+middleFour⇒assoc
-  ; identity+middleFour⇒comm
-  ; assoc+distribʳ+idʳ+invʳ⇒zeˡ
-  ; assoc+distribˡ+idʳ+invʳ⇒zeʳ
-  ; assoc+id+invʳ⇒invˡ-unique
-  ; assoc+id+invˡ⇒invʳ-unique
-  ; comm+distrˡ⇒distrʳ
-  ; comm+distrʳ⇒distrˡ
+  ( comm∧assoc⇒middleFour
+  ; identity∧middleFour⇒assoc
+  ; identity∧middleFour⇒comm
+  ; assoc∧distribʳ∧idʳ∧invʳ⇒zeˡ
+  ; assoc∧distribˡ∧idʳ∧invʳ⇒zeʳ
+  ; assoc∧id∧invʳ⇒invˡ-unique
+  ; assoc∧id∧invˡ⇒invʳ-unique
+  ; comm∧distrˡ⇒distrʳ
+  ; comm∧distrʳ⇒distrˡ
   ; comm⇒sym[distribˡ]
-  ; subst+comm⇒sym
+  ; subst∧comm⇒sym
   ; wlog
   ; sel⇒idem
   )
```

Comment:
We should still be hiding the old deprecated names here. Otherwise, they suddenly appear in scope!

Reply:
Aaargh... I dithered over this, and realised I wasn't quite sure of myself. Thanks for picking this up.

Reply:
I opted to do this in a block, rather than pairing line-by-line. Hope that's OK!

Reply:
A single `hiding` directive is fine!